sections/semantics-embedded-content.include

```diff
@@ -898,13 +899,13 @@
   settings attribute</a>. Its purpose is to allow images from third-party sites that allow
   cross-origin access to be used with <{canvas}>.
 
-  The <dfn element-attr for="img"><code>referrerpolicy</code></dfn> attribute is a <a>referrer policy attribute</a>. Its purpose is to
-  set the <a>referrer policy</a> used when <a>fetching</a> the image. [[!REFERRERPOLICY]]
+  The <dfn element-attr for="img"><code>decoding</code></dfn> attribute is an enumerated attribute.
+  It is an [=image decoding hint=] to request synchronous or asynchronous image loading.
+  Valid values are "<code>async</code>", "<code>async</code>", and "<code>auto</code>".
```

Comment:
`are "<code>async</code>", "<code>async</code>", and` is typo.

Reply:
Oops, sorry, didn't refresh the page before merging... will submit another commit to fix this.

Thanks for the review @mtrootyy .